RealmModelGenerator/DetailsMainVC.swift

```diff
@@ -56,73 +36,43 @@ class DetailsMainVC: NSViewController, EntityDetailVCDelegate, AttributeDetailVC
         detailsContainerView.addSubview(entityDetailVC.view)
         detailsContainerView.addSubview(attributeDetailVC.view)
         detailsContainerView.addSubview(relationshipDetailVC.view)
-        invalidViews()
-        setEmpty()
+        invalidViews(nil)
     }
     
-    func invalidViews() {
+    func invalidViews(viewController: NSViewController?) {
```

Comment:
invalidateViews() is supposed be completely state based and not take any arguments.
if you wish to control which viewController is visible then a value should store the state of the view controllers
then the invalidateViews function should update the hidden state based on the value that the view controller has. This pattern is the exact same as selectedEntity for EntitiesVC